src/TLuaInterpreter.cpp

```diff
@@ -3208,6 +3208,56 @@ int TLuaInterpreter::setLabelClickCallback( lua_State *L )
     return 0;
 }
 
+int TLuaInterpreter::setLabelReleaseCallback( lua_State *L )
+{
+    string luaSendText="";
+    if( ! lua_isstring( L, 1 ) )
+    {
+        lua_pushstring( L, "setLabelReleaseCallback: wrong argument type" );
+        lua_error( L );
+        return 1;
+    }
+    else
+    {
+        luaSendText = lua_tostring( L, 1 );
+    }
+    string luaName="";
+    if( ! lua_isstring( L, 2 ) )
+    {
+        lua_pushstring( L, "setLabelReleaseCallback: wrong argument type" );
+        lua_error( L );
+        return 1;
+    }
+    else
+    {
+        luaName = lua_tostring( L, 2 );
+    }
+
+    TEvent pE;
+
+    int n = lua_gettop( L );
+    for( int i=3; i<=n; i++)
+    {
+        if( lua_isnumber( L, i ) )
+        {
+            pE.mArgumentList.append( QString::number(lua_tonumber( L, i ) ) );
+            pE.mArgumentTypeList.append( ARGUMENT_TYPE_NUMBER );
+        }
+        else if( lua_isstring( L, i ) )
+        {
+            pE.mArgumentList.append( QString(lua_tostring( L, i )) );
+            pE.mArgumentTypeList.append( ARGUMENT_TYPE_STRING );
+        }
+    }
+
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+    QString text(luaSendText.c_str());
+    QString name(luaName.c_str());
+    mudlet::self()->setLabelReleaseCallback( pHost, text, name, pE );
```

Comment:
I removed my own comment somehow? :suspect:

I added the changes to TLabel in the second commit, and the method was added to the mudlet class in the first commit. Never really looked at any of the map stuff, so I have no clue if my changes to TLabel will break things there.